packages/styling/src/styles/DefaultFontStyles.ts

```diff
@@ -130,7 +130,6 @@ function _createFont(size: string, weight: number): IRawStyle {
   return {
     fontFamily: _getFontFamily(),
     MozOsxFontSmoothing: 'grayscale',
-    MsHighContrastAdjust: 'none',
```

Comment:
@mikewheaton I believe these styles were taken directly from fabric core. Is this removal OK?